lib/expression/v2/callbacks.ex

```diff
@@ -109,7 +109,7 @@ defmodule Expression.V2.Callbacks do
       def callback(module \\ __MODULE__, context, function_name, args)
 
       def callback(module, context, built_in, args)
-          when built_in in ["*", "+", "-", ">", ">=", "<", "<=", "/", "^", "=="],
+          when built_in in ["*", "+", "-", "<>", ">", ">=", "<", "<=", "/", "^", "=="],
```

Comment:
Some of our tests use `<>` to join two strings together. I'm not sure if that's standard (I doubt it) but putting it here for backwards compatibility.